lambdas/handlers/manage_nrl_pointer_handler.py

```diff
@@ -58,7 +59,16 @@ def lambda_handler(event, context):
                         .json()
                     )
 
+                    logger.info(
```

Comment:
I notice that the ticket mentions '_When a pointer is created **two** Audit Logs must be made in the system_'

I'll have a chat with Rob (he made the ticket) and work out if there's an intended format for this, but I'd assume we want to create a JSON object and log that out.